packages/libs/avector/README.md

```diff
@@ -0,0 +1,49 @@
+[announcement post]: https://hash.dev/blog/announcing-error-stack
+[crates.io]: https://crates.io/crates/error-stack
+[libs.rs]: https://lib.rs/crates/error-stack
+[rust-version]: https://www.rust-lang.org
+[documentation]: https://docs.rs/error-stack
+[license]: https://github.com/hashintel/hash/blob/main/packages/libs/error-stack/LICENSE.md
+
+[![crates.io](https://img.shields.io/crates/v/avector)][crates.io]
+[![libs.rs](https://img.shields.io/badge/libs.rs-avector-orange)][libs.rs]
```

Comment:
This crate is not published

Reply:
removed in [07829cc](https://github.com/hashintel/hash/pull/1399/commits/07829cc38d3009ccd4bd8329f63976ea7e4e1e66)